README.md

```
@@ -1,435 +1,424 @@
 
-2. **Install dependencies**:
-
-   ```bash
-   # Frontend
-   npm install
-
-   # Backend
-   cd backend
-   npm install
+4. **Access Your App**
    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)